src/liballoc/vec.rs

```diff
@@ -2390,6 +2390,21 @@ impl<T: Clone> From<&mut [T]> for Vec<T> {
     }
 }
 
+#[stable(feature = "vec_from_array", since = "1.42.0")]
```

Comment:
This will not land before `1.44` by now (beta cutoff for `1.42` is some time today). So this should be adjusted to 1.44 right?